### DISCUSSION TITLE
General information about installing on the Electribe

### DISCUSSION THREAD

**Discussion by Maxklui** [bangcorrupt/hacktribe#106] [Q&A] [answered]
Hello guys

I have a general question about installing stuff in the hacktribe firmware. I am quite a noob when it comes to Python and wanted to ask if anyone could give me the basics for installing the e2sysex.py or e2_groove_template. or generally when new content is released for the Electribe. Because I would like to be able to install new content for the Electribe when I publish it here on github.

Thank you in advance and I wish you a nice evening

Greetings Max

**Comment by bangcorrupt** [comment, marked as answer]
Soon there will be a GUI available, [it's being tested now](https://github.com/bangcorrupt/hacktribe/discussions/97).  I will add support for import, export and edit to the GUI over the next few weeks.
